app/providers.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { useState } from "react";
+
+export function Providers({ children }: { children: React.ReactNode }) {
+  const [queryClient] = useState(
+    () =>
+      new QueryClient({
+        defaultOptions: {
+          queries: {
+            staleTime: 60 * 1000, // 1 minute
+          },
+        },
+      }),
+  );
+
+  return <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>;
+}
```

Comment:
Is this plan to switch all of our queries over to this?

Providers is a little generic.  It's also plural.  I would probably rename this to QueryClientProvider or something similar but have it load from the local path.

Shouldn't this live in lib?